packages/inter-protocol/src/vaultFactory/vaultDirector.js

```diff
@@ -469,12 +470,15 @@ const makeVaultDirector = defineDurableExoClassKit(
         const { managerBaggages } = ephemera;
         // alleged okay because used only as a diagnostic tag
         const brandName = await E(collateralBrand).getAllegedName();
+        const collateralUnit = await unitAmount(collateralBrand);
```

Comment:
I don't suppose there's a straightforward way to get the `displayInfo` earlier, is there?